### ISSUE TITLE
Some associated S picks are earlier than P picks at the same station

### ISSUE THREAD

**Issue by zhong-yy** [yetinam/pyocto#12]
Sometimes the S pick is earlier than the P pick at the same station, although it is rare. Are these associated picks wrong?

**Comment by yetinam** [comment]
Yes, these picks are most likely wrong, even though it is not clear which of the picks is wrong. I guess these occur for stations very close to the source? Maybe it would be good to add a constraint to PyOcto for avoiding this case.

**Comment by zhong-yy** [comment]
Thanks for your reply @yetinam . Is this problem because of a large time threshold?

I agree it would be helpful to add a constraint. Alternatively, would it be good to set a distance-dependent threshold or relative threshold?
